sar_pipeline/aws/preparation/config.py

```diff
@@ -6,7 +6,7 @@
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
-VALID_BASE_CONFIGS = ["S1_RTC.yaml","S1_RTC_STATIC.yaml"]
+VALID_BASE_CONFIGS = ["S1_RTC.yaml", "S1_RTC_STATIC.yaml"]
```

Comment:
Given you've now added this, I think it makes sense to update the type hinting of RTCConfigManager to a Literal to help with type checking. Suggested approach is to define the literal first, then build the constant from it:
```python
import typing
from typing import Literal

BaseConfigType = Literal["S1_RTC.yaml", "S1_RTC_STATIC.yaml"]
VALID_BASE_CONFIGS = typing.get_args(BaseConfigType)
```